### PR TITLE
Update log out alert copy

### PR DESCRIPTION
**Pull request by elibud**
**Fixes** #7125 

**To test:**
Me->Log Out
Check that the message and button text is correct.

Needs review: @frosty

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Me/MeViewController.swift [hunk 1]
```diff
@@ -298,7 +298,7 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
 
     fileprivate func confirmLogout() -> ImmuTableAction {
         return { [unowned self] row in
-            let format = NSLocalizedString("Disconnecting your account will remove all of @%@’s WordPress.com data from this device.", comment: "Label for disconnecting WordPress.com account. The %@ is a placeholder for the user's screen name.")
+            let format = NSLocalizedString("Logging out will remove all of @%@’s WordPress.com data from this device.", comment: "Label for disconnecting WordPress.com account. The %@ is a placeholder for the user's screen name.")
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Me/MeViewController.swift]: Maybe update the comment here too?

File: WordPress/Classes/ViewRelated/Me/MeViewController.swift [hunk 2]
```diff
@@ -307,7 +307,7 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
                 style: .cancel,
                 handler: nil)
             let disconnect = UIAlertAction(
-                title: NSLocalizedString("Disconnect", comment: "Button for confirming disconnecting WordPress.com account"),
+                title: NSLocalizedString("Log Out", comment: "Button for confirming disconnecting WordPress.com account"),
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Me/MeViewController.swift]: Maybe update the comment here too?

File: WordPress/Classes/ViewRelated/Me/MeViewController.swift [hunk 3]
```diff
@@ -307,7 +307,7 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
                 style: .cancel,
                 handler: nil)
             let disconnect = UIAlertAction(
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Me/MeViewController.swift]: I'm going to be really nitpicky, but let's rename the action from `disconnect`, too.

**Comment by elibud** [comment]
Hey @frosty you are right about everything, all fixed and ready for another look.

**Comment by frosty** [comment]
Looks great! :shipit:

**Pull request merged**